controls/combobox/styling.md

```diff
@@ -51,37 +53,25 @@ Here is how the styling is applied when the control is focused and item is selec
 
 ## DropDown Styling
 
-The following properties styles the ComboBox Drop Down:
+The following properties style the ComboBox DropDown:
 
-* `DropDownBorderColor`(`Microsoft.Maui.Graphics.Color`): Defines the color of the border around the drop down part of the control.
-* `DropDownBorderThickness`(`Microsoft.Maui.Thickness`): Defines the thickness of the border that is around of the drop down part of the control.
-* `DropDownBorderCornerRadius`(`Microsoft.Maui.Thickness`): Defines the corner radius of the border that is around the drop down part of the control
-* `DropDownBackgroundColor`(`Microsoft.Maui.Graphics.Color`): Defines the background color of the drop down part of the control.
-* `DropDownButtonStyle`(of type `Style` with target type `Telerik.Maui.Controls.RadButton`): Defines the style for the drop down button.
+* `DropDownBorderColor`(`Color`): Defines the color of the border around the drop down part of the control.
```

Comment:
* In the list items, use an em dash as a separator instead of the colon.
* Use either "DropDown" instead of "drop down".